docs/cow-protocol/concepts/flash-loans/integrators.md

```diff
@@ -19,9 +19,13 @@ It is important to ensure that the flash loan gas overhead is added to the slipp
 }
 ````
 
-- **lender (optional):** the contract that could be used to borrow the funds from. For example `0x60744434d6339a6B27d73d9Eda62b6F66a0a04FA` for Maker DAO, or `0x87870Bca3F3fD6335C3F4ce8392D69350B4fA4E2` for AAVE.
-- **borrower (optional):** who should receive the borrowed tokens.
+- **lender (optional):** the contract that could be used to borrow the funds from. For example `0x60744434d6339a6B27d73d9Eda62b6F66a0a04FA` for Maker DAO, or `0x87870Bca3F3fD6335C3F4ce8392D69350B4fA4E2` for AAVE. If the value is not provided the solver will choose a sensible default.
+- **borrower (optional):** who should receive the borrowed tokens. If no value is provided, the order owner's address will be used as the borrower.
 - **token:** the token that needs to get borrowed.
 - **amount:** how many atoms of the token need to get borrowed (e.g., 1 `WETH` would be 10¹⁸).
 
-If optional values were not provided the solver will choose sensible defaults.
+# How a lender protocol can be compatible with CoW Protocol
+
+If the lender protocol rely on direct repayment from the borrower through `transferFrom`, then the lender protocol could implement the borrower functions as in [Aave Borrower](https://github.com/cowprotocol/flash-loan-router/blob/main/src/AaveBorrower.sol) or in [ERC-3156 wrapper](https://github.com/cowprotocol/flash-loan-router/blob/main/src/ERC3156Borrower.sol).
+
+If the lender supports ERC-3156 then there is no extra smart-contract work needed.
```

Comment:
nit: I would flip this and say if the lender is `ERC-3156` compliant no additional SC development is necessary. I hope most protocols are actually compliant and don't do their own thing.